resources/ark/templates/service-monitor.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
   name: {{ template "ark.fullname" . }}
+  namespace: kyma-system
```

Comment:
`namespace:` should be defined as a template. Also can be removed as `helm `take care of the namespaces.

Reply:
ark is installed in a different namespace than Prometheus operator so I need to put the namespace for the service monitor explicitly. Without it the service monitor won't be picked up by the operator.
I extracted the value to the values.yaml